rest_service/Entities/Player.cs

```diff
@@ -1,15 +1,33 @@
 using System.Diagnostics.CodeAnalysis;
 using MongoDB.Bson;
 using MongoDB.Bson.Serialization.Attributes;
+using Newtonsoft.Json;
+using RestService.Dtos.RequestObjects;
 
 namespace RestService.Entities;
 
 [SuppressMessage("ReSharper", "PropertyCanBeMadeInitOnly.Global")]
 public class Player
 {
     [BsonElement("_id")] public ObjectId? Id { get; set; }
-    public string? Nickname { get; set; }
-    public string? TeamName { get; set; }
-    public string? Email { get; set; }
-    [BsonElement("location")] public string? Location { get; set; }
+    [BsonElement("Nickname")] public string? Name { get; set; }
+    [BsonElement("TeamName")] public string? Team { get; set; }
+    [BsonElement("Email")] public string? Email { get; set; }
+
+    [JsonProperty("location")]
+    [BsonElement("location")]
+    public string? Location { get; set; }
+
+    public Player(PlayerRequest request)
```

Comment:
Date Entities must not be aware of DTO's. Ideally the data entities would be distributed in another .net assembly: the Data Layer.